ctapipe/image/muon/intensity_fitter.py

```diff
@@ -536,6 +536,9 @@ def __call__(self, tel_id, center_x, center_y, radius, image, pedestal, mask=Non
         # Perform minimisation
         minuit.migrad()
 
+        # Check for convergence
+        validation = minuit.valid
```

Comment:
This is very nitpicky, but there is no reason to create the local variable here, just do

```
is_valid=minuit.valid
```

below where the container is created.